RELEASING.md

```diff
@@ -139,23 +139,27 @@ Here are the steps:
+1. Since you have 15 minutes free, use that time to update https://github.com/adoptium/adoptium.net/blob/master/src/handlebars/support.handlebars which is the source of  https://adoptium.net/support.html and (if required) the supported platforms table at https://github.com/adoptium/adoptium.net/blob/master/src/handlebars/supported_platforms.handlebars which is the source of https://adoptium.net/supported_platforms.html, and also update https://adoptium.net/release_notes.html ([Sample change](https://github.com/adoptium/adoptium.net/pull/675/commits/563d8e2f0d9d26500a7e8d9eca61b491f73f1f37)
+1. [Mac only] Once the binaries are available on the website you need to update the Homebrew casks. There are 3 casks in total ([`temurin`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/temurin.rb) which always serves the latest version, [`temurin8`](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/temurin8.rb) and [`temurin11`](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/temurin11.rb)). An example PR can be found [here](https://github.com/Homebrew/homebrew-cask/pull/113581). If in doubt reach out to George (gdams) as he's a maintainer.
+1. [Linux only] Once the binaries are available on the website you can begin updating the specfiles for the RPM/DEB files. There are 3 different type of linux installer ([debian](https://github.com/adoptium/installer/tree/master/linuxNew/jdk/debian/src/main/packaging/temurin), [redhat](https://github.com/adoptium/installer/tree/master/linuxNew/jdk/redhat/src/main/packaging/temurin) and [suse](https://github.com/adoptium/installer/tree/master/linuxNew/jdk/suse/src/main/packaging/temurin)) and all 3 need updating. Once these files have been updated the [adoptium-packages-linux-pipeline](https://ci.adoptopenjdk.net/job/adoptium-packages-linux-pipeline/) job needs to be kicked off making sure to tick the `UPLOAD` parameter.
+  - **Debian** you need to locate the `rules` file for each version. This file contains the URLs and Checksums for each package ([example](https://github.com/adoptium/installer/blob/master/linuxNew/jdk/debian/src/main/packaging/temurin/11/debian/rules#L6-L15)).
+  - **RedHat/Suse** you need to modify the `temurin-<version>-jdk.spec` file for each version. The links/checksum links are all defined as `Source<number>` variables ([example](https://github.com/adoptium/installer/blob/master/linuxNew/jdk/redhat/src/main/packaging/temurin/11/temurin-11-jdk.spec#L128-L142)).
+1. [Docker] please see https://github.com/adoptium/containers#maintenance-of-dockerfiles
```

Comment:
I think the formatting will need some fixup here as the two previous bullet points have caused this to show as step 1 again.